modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -19,6 +19,7 @@ const defaultProps = {
   data: null,
   loader: Tiles3DLoader,
 
+  pickFeatures: {type: 'boolean', value: false},
```

Comment:
Isn't this prop removed per our discussion in loaders.gl? (Always pick features if ids are available)

Reply:
Yes the final decision is to remove `pickFeatures` flag. Done